code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1630,3 +1630,14 @@
 	description = "blue sparkles that get everywhere"
 	color = "#4040FF" //A blueish color
 	glitter_type = /obj/effect/decal/cleanable/glitter/blue
+
+/datum/reagent/pukonium
```

Comment:
Yeah I know, I mean that I don't want disgust to be something only caused by bad cooking. I also see disgust as something that can be caused by seeing disgusting things or having something gross happen to you, pukonium is a bit more general in that sense.